action.yml

```diff
@@ -12,4 +16,4 @@ runs:
   using: "composite"
   steps:
     - shell: bash
-      run: ${{ github.action_path }}/run.sh ${{ inputs.directory }} ${{ inputs.release-branch-prefix }}
+      run: ${{ github.action_path }}/run.sh ${{ inputs.directory }} ${{ inputs.release-branch-prefix }} ${{ inputs.npm-build-command }}
```

Comment:
I think it should be quoted here too, to prevent word splitting. Otherwise you could chain more arbitrary commands from here.